src/button/cta.scss

```diff
@@ -6,12 +6,16 @@
 @import './disabled';
 
 @mixin iui-button-cta {
+  --_iui-button-label-color: var(--iui-color-foreground-accessory);
+  --_iui-button-label-color-hover: --_iui-button-label-color;
+
+  @include iui-focus($color: var(--iui-color-foreground-accessory), $offset: -3px, $thickness: 1px);
```

Comment:
Can we not move the `iui-focus` around for no reason? From an organizational and cascade perspective, also doesn't make sense to have modifiers (i.e. `:focus`/`:focus-visible`) before the base styling

Reply:
Moved the `iui-focus` mixin back to where it was originally.